index.js

```diff
@@ -43,16 +54,58 @@ app.get("/milestones", (req, res) => {
   res.render("milestones");
 });
 
+// GET /login - returns the login page
 app.get("/login", (req, res) => {
-  res.render("login");
+  // Render the login view. Provide no error to the view.
+  res.render("login", { error: "" });
 });
 
-app.post("/login_submit", (req, res) => {
-  res.render("login_submit");
+// POST /login - validates the submitted form data
+app.post("/login", (req, res) => {
+  const email = req.body.email;
+  const password = req.body.password;
+
+  console.log("form submitted", email, password);
+  getUser(
+    email,
+    (user) => {
```

Comment:
Because this is the 2nd parameter of `getUser`, it means that this function is being passed in to `getUser` and will be executed when the query is complete.

The `getUser` provides us the query result (`res.rows`)